temporal-serviceclient/src/main/java/io/temporal/serviceclient/RpcRetryOptions.java

```diff
@@ -376,6 +409,9 @@ public void validate(boolean hasToBeFinite) {
     if (initialInterval == null) {
       throw new IllegalStateException("required property initialInterval not set");
     }
+    if (longInitialInterval == null) {
```

Comment:
@Spikhalskiy Not sure what's currently the preferred way to add an option to an existing Option object.

There is probably applications out there that use `build()` instead of `validateBuildWithDefaults()`, which means that at some point, there is a risk that either the new option is null, or the option object provided by the user is used to create a new builder which is `validated()` (I have seen a few occurences of this). How do you usually deal with this?

Reply:
We shouldn't make this property required. Too nuanced. Either attach it to a value of `initialInterval` with some reasonable multiplier or give it a default value. Users shouldn't fight their way so hard to create a functional instance.